packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -32,22 +29,18 @@ export async function createServer(
   logger: Logger,
   context?: () => object
 ): Promise<ApolloServer> {
-  setupDatabase(config.database);
-  const fetchUserFn = Container.get(userLookupFnContainerToken);
   const schema = await TypeGraphQL.buildSchema({
     container: Container,
     resolvers: [AccountResolver],
   });
   const debugMode = config.env !== 'production';
   const defaultContext = async ({ req }: any) => {
     const bearerToken = req.headers[config.authHeader.toLowerCase()];
-    const authUser = await fetchUserFn(bearerToken);
-    if (!authUser) throw new AuthenticationError('You must be logged in');
-
+    const userId = bearerToken.split('Bearer ')[1];
```

Comment:
The config value for `authHeader` is a string, so it might not have `Bearer` in it. Could do a string replace or update doc to mention it.

Reply:
True, this is a temporary hack. As the auth method didn't pan out, this is temporary for local testing. The next issue I have is to get in proper `sessionToken` support which will replace this.